File: Java/UberX.java
package Java;

public class UberX extends UberCar {
    String brand;
    String model;

    public UberX(String license, Account driver, Integer passenger, String brand, String model){
        //Use super to pass parameter to subclass (Using Inheritance)
        super(license, driver, passenger);
        this.brand = brand;
        this.model = model;
    }
    
}

File: Python/uberX.py
from uberCar import UberCar

class UberX(UberCar):
    #This way to use Inheritance
    brand = str
    model = str

    def __init__(self,license,driver,brand,model):
        #Super class to inheritance. Use super()
        super().__init__(license,driver)
        self.brand = brand
        self.model = model